tests/end_to_end/test_flag_engine_end_to_end.py

```diff
@@ -1,82 +0,0 @@
-from datetime import datetime
-
-from flag_engine.api.document_builders import (
-    build_environment_document,
-    build_identity_document,
-)
-from flag_engine.engine import (
-    get_environment_feature_states,
-    get_identity_feature_states,
-)
-from flag_engine.environments.builders import build_environment_model
-from flag_engine.identities.builders import build_identity_dict, build_identity_model
-from flag_engine.identities.traits.models import TraitModel
-from tests.mock_django_classes import DjangoFeatureState, DjangoIdentity
-
-
-def test_environment_end_to_end(mock_django_environment):
-    # First, we need to dump the environment as if we were sending it to DynamoDB
-    environment_dict = build_environment_document(mock_django_environment)
-    assert environment_dict
-
-    # Then we should be able to load that data again as if we were grabbing the
-    # environment from dynamodb in the lambda edge functions
-    environment_model = build_environment_model(environment_dict)
-    assert environment_model
-
-    # Finally, we should be able to get the flags for this environment
-    feature_states = get_environment_feature_states(environment_model)
-    # and the feature states should be correct
-    assert len(feature_states) == 1
-    assert feature_states[0].enabled is True
-    assert feature_states[0].get_value() == "foobar"
-
-
-def test_identity_end_to_end(mock_django_environment, mock_django_feature):
```

Comment:
It makes me a little nervous throwing away these tests. Are they covered in the Core API PR perhaps?

Reply:
Indeed these cases are covered in the Core API PR, and we should have more confidence in them because they use real Django model instances.